File: tasks/implementations/java/tasks/api_client.java
import java.nio.file.*;
import java.util.regex.*;

class ApiClient {
    public static void run(String size, String fixturesRoot) throws Exception {
        String text = Files.readString(Path.of(fixturesRoot, "mocks", "public_api_" + size + ".json"));
        Matcher matcher = Pattern.compile("\\\"id\\\":\\s*(\\d+),\\s*\\\"name\\\":\\s*\\\"([^\\\"]+)\\\",\\s*\\\"value\\\":\\s*(\\d+)").matcher(text);
        long total = 0;
        while (matcher.find()) total += Integer.parseInt(matcher.group(1)) + Integer.parseInt(matcher.group(3)) + matcher.group(2).length();
        System.out.println(total);
    }
}

File: tasks/implementations/python/tasks/api_client.py
from pathlib import Path
import re


def run(size: str, fixtures_root: str) -> None:
    text = (Path(fixtures_root) / "mocks" / f"public_api_{size}.json").read_text(encoding="utf-8")
    total = 0
    for item_id, name, value in re.findall(r'"id":\s*(\d+),\s*"name":\s*"([^"]+)",\s*"value":\s*(\d+)', text):
        total += int(item_id) + int(value) + len(name)
    print(total)
